index.js

```diff
@@ -4,7 +4,12 @@ const isPng = require('is-png');
 const optipng = require('optipng-bin');
 
 module.exports = opts => buf => {
-	opts = Object.assign({optimizationLevel: 3}, opts);
+	opts = Object.assign({
+		optimizationLevel: 3,
+		disableBitDepthReduction: false,
+		disableColorTypeReduction: false,
+		disablePaletteReduction: false
```

Comment:
You could remove these since they're `undefined` by default and therefore won't be added.